linters/djlint/.djlintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "linter_output_format": "{filename}:{line} - {code} - {message} :: {match}"
```

Comment:
Probably don't need {match} here, since it looks like all of these are optional. If you suspect it provides additional value, lmk. Otherwise, I really appreciate having this output format option.

Reply:
Match is super useful here as it describes what the error is talking about.
test_data/logout.html:7:4 - H017 - Tag should be self closing. <link>
 '<link>'  is the match for this example. 

Rendered in Trunk as 

```
**test_data/logout.html:7:4**
 7:4  high  Tag should be self closing. :: <link>  **djlint/H017**
```